datajunction-server/datajunction_server/api/helpers.py

```diff
@@ -849,7 +853,7 @@ def build_sql_for_multiple_metrics(  # pylint: disable=too-many-arguments,too-ma
                     ColumnMetadata(name=col.name, type=str(col.type))
                     for col in (metric_columns + dimension_columns)
                 ],
-                dialect=engine.dialect,
+                dialect=materialized_cube_catalog.engines[0].dialect,
```

Comment:
With Druid as the only engine for cubes this is ok. But we should probably make a comment here that this is a temporary shortcut.